### DISCUSSION TITLE
3 Wide Layout for Ultrawide Users.

### DISCUSSION THREAD

**Discussion by Ecl1pseGST** [Garux/netradiant-custom#120] [Ideas]
Would it be possible to get a layout where the toolbar is on the far right, 3D in middle, 2D on left? Example from Radiant Black (CoD BO3 Editor) attached
![Screenshot (850)](https://user-images.githubusercontent.com/102322421/200032170-08a036e6-b0c5-42b1-adc1-f7d9000bb1d4.png)

**Comment by Garux** [comment]
Looks doable and making good sense.

**Comment by Garux** [comment]
Added option for this https://github.com/Garux/netradiant-custom/commit/e1c814f600cabf83b2e7839431b9457932bb1c4d
